tutorials/join-across-data-sources-with-dbt.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Join Across Data Sources with GlareDB and dbt
+layout: default
+nav_order: 3
+parent: Tutorials
+---
+
+# Using dbt with GlareDB
+
+GlareDB can serve as a destination warehouse for dbt, allowing you to build
+transformation pipelines that work with data from multiple sources without
+additional ETL infrastructure, so you can save money and time by not needing
+to configure data extraction with tools like Fivetran or Stitch.
+And since GlareDB can connect using the Postgres protocol, you can use the
+existing Postgres adapter without needing anything new. This guide
+will walk you through setting up and using dbt with GlareDB.
+
+## Prerequisites
+
+- dbt Core installed (`pip install dbt-postgres`)
+- A GlareDB Cloud account (sign up at [console.glaredb.com])
+- Basic familiarity with dbt concepts
+
+## Setup
+
+### 1. Initialize a dbt Project
+
+Create a new dbt project:
+
+```shell
+dbt init my_glaredb_project
+```
+
+### 2. Configure dbt Credentials
+
+Add your GlareDB credentials to your `profiles.yml` file. You can find your
+connection details in GlareDB Cloud by clicking the "Connect" button and selecting
+the Postgres tab.
+
+```yaml
+dbt_glaredb_quickstart:
+  outputs:
+    dev:
+      dbname: org_name/deployment_name
+      host: proxy.glaredb.com
+      pass: your_password
+      port: 6543
+      schema: public
+      threads: 1
+      type: postgres
+      user: your_username
+  target: dev
+```
+
+> **Note**: When using your GlareDB Cloud connection string (which looks like
+> `glaredb://user_name:password@org_name.proxy.glaredb.com:6543/deployment_name`),
+> specify your `dbname` as `org_name/deployment_name` and the `host` as
+> `proxy.glaredb.com`.
+
+### 3. Test Your Connection
+
+Verify your connection:
+
+```shell
+dbt debug
+```
+
+## Creating dbt Models
+
+### Basic Model Example
```

Comment:
the fact that this doesn't syntax highlight correctly, makes me think that there's a syntax error probably somewhere between 55 and 60, but maybe not.